pkgs/applications/audio/sonixd/default.nix

```diff
@@ -19,9 +19,9 @@ appimageTools.wrapType2 rec {
     mv $out/bin/${pname}-${version} $out/bin/${pname}
 
     install -m 444 -D ${appimageContents}/${pname}.desktop -t $out/share/applications
-
-    install -m 444 -D ${appimageContents}/${pname}.png \
-      $out/share/icons/hicolor/1024x1024/apps/${pname}.png
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
```

Comment:
Noticed there's a `--no-sandbox %U` option in the original desktop file that we could remove also I guess ?

It doesn't pose any issue to leave it though

Reply:
You're right! Thank you I think it is fixed now.